spec.html

```diff
@@ -8258,7 +8258,6 @@
 
       <emu-clause id="sec-set-immutable-prototype">
         <h1>SetImmutablePrototype ( _O_, _V_ )</h1>
-        <!-- This is used by host environment specs, even though it only has one consumer in this spec; don't inline it -->
```

Comment:
Either way tho, I think the comment is an important marker for the external dependency, and I'd expect every abstract operation that's referenced externally to be similarly noted.